doc/rdoc/markup_reference.rb

```diff
@@ -7,72 +7,74 @@
 # attributes, and constants -- are solely for illustrating \RDoc markup,
 # and have no other legitimate use.
 #
-# = \RDoc Markup Reference
-#
-# Notes:
+# == About the Examples
 #
 # - Examples in this reference are Ruby code and comments;
 #   certain differences from other sources
 #   (such as C code and comments) are noted.
+# - Almost all examples on this page are all RDoc-like;
+#   that is, they have no explicit comment markers like Ruby <tt>#</tt>
+#   or C <tt>/* ... */</tt>.
 # - An example that shows rendered HTML output
 #   displays that output in a blockquote:
 #
-#   Rendered HTML:
 #   >>>
 #     Some stuff
 #
 # \RDoc-generated documentation is derived from and controlled by:
```

Comment:
Did you mean to remove this line too since the bullet points below are removed?

Reply:
Removed.